default.vcl

```diff
@@ -61,6 +61,8 @@ sub vcl_recv {
         # Routing preset here as vulcan is unable to route on query strings
     } elseif (req.url ~ "\/content\?.*isAnnotatedBy=.*") {
         set req.http.Host = "public-content-by-concept-api";
+    } elseif (req.url ~ "\/concept\/search.*$") {
```

Comment:
Oops, realised I was wrong about the second part as missed out the `\?` in the regex for `isAnnotatedBy`.